west.yml

```diff
@@ -1,29 +1,6 @@
 manifest:
   projects:
-    - name: zephyr
-      revision: d51a67b7c098c8bb5479a7b87f05dc40fb76e065
-      url: https://github.com/zephyrproject-rtos/zephyr
-      west-commands: scripts/west-commands.yml
-      import:
-        name-allowlist:
-          - cmsis
-          - hal_espressif
-          - hal_nordic
-          - hal_st
-          - hal_stm32
-          - mbedtls
-          - mcuboot
-          - mcumgr
-          - net-tools
-          - segger
-          - tinycbor
-          - tinycrypt
-
     - name: qcbor
       revision: 17b5607b8c49b835d22dec3effa97b25c89267b3
       url: https://github.com/golioth/QCBOR.git
       path: modules/lib/qcbor
-
-  self:
-    path: modules/lib/golioth
-    west-commands: scripts/west-commands.yml
```

Comment:
>What about naming this to west-external.yml (or similar)?
>
>Most people will expect a file named west.yml to include more than just a stray dependency like this, so I'm mostly trying to avoid confusion.

Got the same concerns. Updated PR to use `west-external.yml`.